docs/reference/3-admin-api/installation/admin-api-1x-for-odsapi-5x-6x/admin-api-1x-iis-installation-powershell.md

```diff
@@ -0,0 +1,417 @@
+> }
+> $adminApiFeatures = @{
+>     ApiMode = "sharedinstance"
+> }
+> $authenticationSettings = @{
+>     Authority = "[https://localhost/adminapi](https://localhost/adminapi)"
+>     IssuerUrl = ""[https://localhost/adminapi](https://localhost/adminapi)"
+>     SigningKey = "Base64-encoded string"
+>     AllowRegistration = $false
+> }
+> $p = @{
+>     ToolsPath = "C:/temp/tools"
+>     DbConnectionInfo = $dbConnectionInfo
+>     OdsApiUrl = "[http://web-api.example.com/WebApi](http://web-api.example.com/WebApi)"
+>     PackageVersion = '1.4.0.0'
+>     PackageSource = $adminApiSource
+>     AuthenticationSettings = $authenticationSettings
+>     AdminApiFeatures = $adminApiFeatures
+> }
+> ```
+>
+> ![Screenshot](https://edfi.atlassian.net/wiki/images/icons/grey_arrow_down.png)
+>
+> PostgreSQL Server
+>
+> ```
+> $dbConnectionInfo = @{
+>         Server = "localhost"
+>         Engine = "PostgreSQL"
+>         UseIntegratedSecurity = $false
+>         Username = "postgres"
+>         Password = "examplePassword"
+> }
+> $adminApiFeatures = @{
+>     ApiMode = "sharedinstance"
+> }
+> $authenticationSettings = @{
+>     Authority = "[https://localhost/adminapi](https://localhost/adminapi)"
+>     IssuerUrl = ""[https://localhost/adminapi](https://localhost/adminapi)"
+>     SigningKey = "Base64-encoded string"
+>     AllowRegistration = $false
+> }
+> $p = @{
+>     ToolsPath = "C:/temp/tools"
+>     DbConnectionInfo = $dbConnectionInfo
+>     OdsApiUrl = "[http://web-api.example.com/WebApi](http://web-api.example.com/WebApi)"
+>     PackageVersion = '1.4.0.0'
+>     PackageSource = $adminApiSource
+>     AuthenticationSettings = $authenticationSettings
+>     AdminApiFeatures = $adminApiFeatures
+> }
+> ```
+
+### **Step 3. Open a PowerShell Prompt in Administrator Mode**
+
+Method 1: Open \[Windows Key\]-R which will open a Run dialog for tasks needing
+administrative privileges. Type "PowerShell" to open a PowerShell prompt in
+Administrator mode.
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v1/image2020-4-20_12-37-43.png)
+
+Method 2: Click on the Windows icon in the lower-left corner. Type "PowerShell"
+and right-click the "Windows PowerShell" option when provided. Select "Run as
+Administrator" to open a PowerShell prompt in Administrator mode.
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v1/image2020-4-20_12-37-57.png)
+
+Change the directory to the unzipped directory for the Admin Api Installer.
+
+### **Step 4 .** **Run the Installation via PowerShell**
+
+Run "install.ps1" script.
+
+### Database login setup on integrated security mode
+
+During the installation process, you will be prompted to choose database login
+details. Entering "Y" will continue with default option (Installation process
+will create IIS APPPOOL\\AdminApi database login on the server).
```

Comment:
To my surprise, this double backslash renders properly as a single backslash. All is well.